org.messerlab.slimgui.yml

```diff
@@ -0,0 +1,44 @@
+app-id: org.messerlab.slimgui
+runtime: org.kde.Platform
+runtime-version: '5.15-22.08'
+sdk: org.kde.Sdk
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --device=dri
+  - --filesystem=home
+  - --talk-name=com.canonical.AppMenu.Registrar # required for global menu
+  - --talk-name=org.freedesktop.Flatpak # allows spawning processes on the host via flatpak-spawn --host
+  - --env=PKG_CONFIG_PATH=/app/lib/pkgconfig:/app/share/pkgconfig
+command: SLiMgui
+modules:
+  - name: SLiMgui
+    builddir: true
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DBUILD_SLIMGUI=ON
+    sources:
+      - type: git
+        url: https://github.com/messerlab/slim.git
+        tag: v4.0.1
+        commit: 09fcf604a06307cf75098dfa0a2171a9b7852216
+    post-install:
+      # Colourful and Symbolic Application Icons
+      # This file is modified to produce a symbolic icon that works well with GNOME 3; the changes haven't been integrated upstream yet.
```

Comment:
tbf we don't care about a symbolic icon. So if it's not provided by upstream there is no point of doing all of this.

Reply:
A symbolic icon is not exported by Flatpak?